data/human/human missions.txt

```diff
@@ -7453,3 +7453,405 @@ mission "Adelita 5"
+			`	"In fact, the ancillary benefits are largely why there's such discretion in how the expansion will go. The aggressive and conservative fueling options both offer R&D and partnership benefits that are more than the project's direct returns."`
+			choice
+				`	"That makes sense. Let's discuss the deal more after Oblate."`
+				`	"Thanks for the overview, but I'm still not interested. I'm out."`
+					decline
+			label accepting
+			`	Howard nods with approval. "Excellent. The Marauder ship was last seen near Oblate, in the outer rim of that system. It's likely to be well armed, so stay sharp. Once you've disabled it, head back here and I'll make sure you're properly compensated."`
+			branch blewupgolden
+				has "syndicate business golden jewel dead"
+			`	He cracks a smile. "I remember the first time we met you left that Vanguard alive. Same deal here, but disable the Marauder. Hopefully that Vanguard's not still around.`
+				goto ending
+			label blewupgolden
+			`	He leans in slightly, lowering his voice. "Just be careful. We don't want to destroy the ship. Cripple it so we can study it, but don't blow it into oblivion like you did to that Vanguard when we first met.`
+			label ending
+			`	"Well, that's all I have for you. Goodbye and good luck!"`
+				accept
+
+	on enter "Moktar"
+		"reputation: Independent (Killable)" = -1000
+	npc
+		government "Independent (Killable)"
+		personality heroic staying vindictive
+			confusion 20
+		system "Moktar"
+		ship "Vanguard" "Golden Jewel"
+		to despawn
+			has "syndicate business golden jewel dead"
+		on encounter
+			conversation
+				`When you enter Moktar, your sensors detect the same pirate Vanguard from when you first rescued Howard. Your ship is immediately hailed, and the pirate opens communication.`
+				`	"I should have destroyed you when I had the chance. Prepare to meet your end!"`
+				choice
+					`	"Maybe I should have destroyed you!"`
+					`	"Uhhh... who are you again?"`
+						goto whomst
+				`	"Argh, you slithering, spineless snake! I'll tear your ship apart piece by piece!"`
+					goto power
+				label whomst
+				`	"Bah, you banal, bumbling baboon! Prepare to meet your doom!"`
+				label power
+				`	You can't help feeling a sense of deja vu as the Vanguard's engines roar to life and you see over half a dozen fixed guns take aim at your ship, ready to strike.`
+		on kill
+			set "syndicate business golden jewel dead"
+			dialog "The crew of the Golden Jewel are no more. You think you can make out the pirate captain drifting in space nearby."
+
+	on enter "Oblate"
+		"reputation: Independent (Killable)" = -1000
+	npc disable save
+		government "Independent (Killable)"
```

Comment:
I'm not sure it makes sense for the Marauder to be Independent. Maybe not even the Vanguard, but that's a side trail...
Then again, if the player is happy with pirates, this could run into trouble. We really need a `Pirate (Killable)` government.
```suggestion
		government "Pirate"
```

Reply:
Hmmm, making them a straight `"Pirate"` government makes things a bit hairy if the player is friendly with pirates as you mentioned and takes away player agency if a third party like the Republic/Syndicate shows up and fails either mission for the player by killing the ship. A `Pirate (Killable)` gov would fix that, but without it I have those concerns. Do you still think it makes sense to change this to Pirate, or leave as `"Independent (Killable)"`?

Reply:
That's what I was wondering.
Probably leave it as Independent for now than - the `Pirate (Killable)` government being added has a separate issue already.